### PR TITLE
Default Branch update: Update master branch to main.

### PR DESCRIPTION
**Pull request by azabbasi**

### CODE REVIEW

File: tools/diffscripts/diffapi.ps1 [hunk 1]
```diff
@@ -302,15 +302,15 @@ Set-Location -Path $compareDirectory
...
-# 9       3       sdk_design_docs/CSharp/master/Microsoft.Azure.Devices.Client.md
-# 2       0       sdk_design_docs/CSharp/master/Microsoft.Azure.Devices.Provisioning.Client.md
-# 2       0       sdk_design_docs/CSharp/master/Microsoft.Azure.Devices.Provisioning.Security.Tpm.md
-# 2       0       sdk_design_docs/CSharp/master/Microsoft.Azure.Devices.Provisioning.Service.md
-# 2       0       sdk_design_docs/CSharp/master/Microsoft.Azure.Devices.Provisioning.Transport.Amqp.md
-# 2       0       sdk_design_docs/CSharp/master/Microsoft.Azure.Devices.Provisioning.Transport.Http.md
-# 2       0       sdk_design_docs/CSharp/master/Microsoft.Azure.Devices.Provisioning.Transport.Mqtt.md
-# 2       0       sdk_design_docs/CSharp/master/Microsoft.Azure.Devices.Shared.md
-# 7       9       sdk_design_docs/CSharp/master/Microsoft.Azure.Devices.md
+# 9       3       sdk_design_docs/CSharp/main/Microsoft.Azure.Devices.Client.md
```
Comment (azabbasi) [inline tools/diffscripts/diffapi.ps1:305]: these are already changed in iot-sdk-internals repository

**Review by jamdavi — APPROVED** [review]

**Pull request merged**